package TryCatch.Chaining;

public class DAOExeption extends Exception{
    public DAOExeption(String message, Throwable cause) {
        super(message, cause);
    }
}
